soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series

```diff
@@ -40,6 +40,10 @@ config IT8XXX2_PLL_SEQUENCE_PRIORITY
 	default 1
 	depends on SOC_IT8XXX2_PLL_FLASH_48M
 
+config IT8XXX2_VCMP_INIT_PRIORITY
+	int
```

Comment:
You don't need set the Kconfig type here.

Reply:
I tried remove "int" type and west build, then it showed "warning: IT8XXX2_VCMP_INIT_PRIORITY defined without a type".

Reply:
That's because it isn't defined in `drivers/sensor/ite_vcmp_it8xxx2/Kconfig`

Reply:
Done